.circleci/src/commands/install_app_toolbelt.yml

```diff
@@ -7,4 +7,4 @@ steps:
         echo 'export PATH=~/.npm-global/bin:"$PATH"' >> "$BASH_ENV"
   - run:
       name: Install app-toolbelt
-      command: npm install --unsafe-perm -g https://github.com/digitalfabrik/app-toolbelt/archive/refs/heads/main.tar.gz
\ No newline at end of file
+      command: npm install --unsafe-perm -g https://github.com/digitalfabrik/app-toolbelt/archive/refs/tags/v0.2.tar.gz
```

Comment:
```suggestion
      command: npm install --unsafe-perm -g https://github.com/digitalfabrik/app-toolbelt/archive/refs/tags/v0.2.tar.gz

```

Reply:
Why we need an empty line?

Reply:
We use to have an empty line at the end of files. But optional since we don't enforce it for .yml files.